app/routes.js

```diff
@@ -349,10 +349,6 @@ const routes = (
       queries={{ content: planQueries }}
       prepareParams={preparePlanParams}
       render={{ content: SummaryPageWrapper }}
-      loadAction={params => [
```

Comment:
https://github.com/HSLdevcom/digitransit-ui/blob/8f6a03aff5afa02813e680a05138df52cca1801e/app/server.js#L240-L245 Could also be removed